simplq/src/components/pages/Admin/index.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
Added this for the onClicks on the divs in the navbar. This is a requirement for accessibility. I was thinking we'd decide what to do here after some thought. https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md